ru_ru/core.standart.json

```diff
@@ -0,0 +1,8 @@
+[
+	"На основе кремниевых элементов и медных деталей в качестве соединительного звена, а также используяOn the basis of the silicon elements and copper parts as connectors, as well as the usage of iron for contamination of the silicon wafer, I managed to create a universal processing unit.",
+	"With correct outer controlling, my primitive processor can go through algorithms and perform many different calculations.",
```

Comment:
Is there a reasons why only a part of this was translated?

Reply:
Sorry, I forgot to upload the latest version of this file. Now it's right.